components/shared_code/src/shared/model/measurement.py

```diff
@@ -302,6 +302,8 @@ def summarize_latest(self) -> Self:
             if parameter_hash != self.metric.source_parameter_hash():
                 self["outdated"] = True
             del self["source_parameter_hash"]
+        if measurement_requested := self.metric.get("measurement_requested"):
+            self["measurement_requested"] = measurement_requested >= self["end"]
```

Comment:
Should we also add an entry to the audit log for this action?

Reply:
There is, the frontend uses the `/api/internal/metric/<metric_uuid>/attribute/<metric_attribute>` endpoint with `metric_attribute=measurement_requested` to record the timestamp of the request. So it's logged the same as other metric attributes.